src/commands/misc/helpRemember.js

```diff
@@ -1,44 +1,56 @@
-const { EmbedBuilder } = require('@discordjs/builders');
+const { SlashCommandBuilder, EmbedBuilder } = require('@discordjs/builders');
 
 module.exports = {
-    name: 'help-remember',
-    description: 'List of commands for remembering messages',
+    data: new SlashCommandBuilder()
+    .setName('help-remember')
+    .setDescription('List of commands for remembering messages'),
 
-    callback: (client, interaction) => {
+    options:
+    {
+        devOnly: false,
+        testOnly: false,
+        deleted: false
+    },
+
+    async execute(client, interaction){
         //Create an embed to send to the user
         const embed = new EmbedBuilder()
             .setTitle('Remember Commands')
             .setDescription('Commands relating to message storage and retrieval')
             .addFields(
                 {
                 // name - /[name of command]
-                    name: '/remember',
+                    name: '/remember message',
                     // value - description of command
                     value: 'Stores a message from the current channel based on its message-id'
                 },
                 {
-                    name: '/clear-messages',
+                    name: '/remember clear',
                     value: 'Clear all messages currently saved in remembrance'
                 },
```

Comment:
The name of the actual command is `/remember clear-messages`. It's fine either way, but needs to be consistent